Toggl.Daneel/ViewSources/MainTableViewSource.cs

```
@@ -20,10 +21,16 @@ public IMvxAsyncCommand<TimeEntryViewModel> ContinueTimeEntryCommand
             set => timeEntriesLogViewSource.ContinueTimeEntryCommand = value;
         }
 
+        public NestableObservableCollection<TimeEntryViewModelCollection, TimeEntryViewModel> ObservableCollection
+        {
+            get => timeEntriesLogViewSource.ObservableCollection;
+            set => timeEntriesLogViewSource.ObservableCollection = value;
+        }
+
         public override IEnumerable ItemsSource
         {
             get => timeEntriesLogViewSource.ItemsSource;
-            set => timeEntriesLogViewSource.ItemsSource = value;
+            set { throw new InvalidOperationException($"You must bind to the {nameof(ObservableCollection)} and not the {nameof(ItemsSource)}"); }
```

This makes me happy

---

Why? (curious)

---

Because that was a very big concern of mine: people binding to the wrong property

---

This is still unfortunately only a silent error - MvvmCross catches it and puts it into the application output. But I hope that it is still better than nothing.